marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/updateMergeOptionsLib.sjs

```diff
@@ -177,7 +177,7 @@ function mergeRule(item, algorithms, properties, namespaces) {
     }
     mr.priorityOrder = priorityOrder;
   }
-
+/*
```

Comment:
Will this be needed later? I generally am in favor of deleting commented code and not leaving it around, but if the assumption is that we'll need it very soon, it's fine to leave it in place.

Reply:
I meant to take that out...